apps/console/src/features/applications/components/wizard/oauth-protocol-settings-wizard-form.tsx

```diff
@@ -495,10 +495,8 @@ export const OauthProtocolSettingsWizardForm: FunctionComponent<OAuthProtocolSet
                                                     inboundOIDC.fieldscallBackUrls.info"
                                                     tOptions={ { callBackURLFromTemplate: callBackURLFromTemplate  } }
                                                     >
-                                                        This field is required for a functional app. 
-                                                        However, if you are planning to 
-                                                        try out the sample app, 
-                                                        you can use <strong>{ callBackURLFromTemplate }</strong>.
+                                                        Don’t have an app? Try out the sample app
```

Comment:
`...Try out the sample app...` -> `... Try out a sample app ....`

Reply:
Fixed